runtime/src/comm/gasnet/comm-gasnet.c

```diff
@@ -340,6 +390,32 @@ static void AM_bcast_seginfo(gasnet_token_t token, void *buf, size_t nbytes) {
   bcast_seginfo_done = 1;
 }
 
+// Put from arg->src (which is local to the AM handler) back to
+// arg->dst (which is local to the caller of this AM).
+// nbytes must be < gasnet_AMMaxLongReply.
```

Comment:
Ah, I misread the comment as if it were expressing a requirement rather than a truth. Maybe rephrase as something like "nbytes is < gasnet_AMMaxLongReply here (see chpl_comm_get)."